api/controllers/game/join-rematch.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Join the two players into the new game
+ */
+module.exports = async function (req, res) {
+  try {
+    const { usr: userId, game: gameId } = req.session;
+    const user = await User.findOne({ id: userId });
+    const oldGame = await Game.findOne({ id: gameId });
+    const newGameId = oldGame.rematchGame;
+    const game = await Game.findOne({ id: newGameId }).populate('players');
+    Game.subscribe(req, [game.id]);
+
+    req.session.game = game.id;
+    req.session.pNum = user.pNum;
+
+    const gameUpdates = {
+      lastEvent: { change: 'joinRematch' },
+    };
+    await Game.updateOne({ id: game.id }).set(gameUpdates);
+    Game.publish([game.id], {
+      change: 'joinRematch',
+      game,
```

Comment:
This endpoint is used after the cards are dealt right? I think we'll want to populate the game that's sent in the socket message so it has the full game data